app/assets/javascripts/services/project_roles_svc.js

```diff
@@ -0,0 +1,28 @@
+samson.service('projectRolesService', function ($http) {
+
+    this.loadProjectRoles = function () {
+        return $http.get('/project_roles');
+    };
+
+    this.createProjectRole = function (project_role) {
+        var data = {
+            project_role: {
+                user_id: project_role.user_id,
+                project_id: project_role.project_id,
+                role_id: project_role.role_id
+            }
+        };
```

Comment:
I'll be curious to see how it turns out! :)  As long as the code is readable, and doesn't needlessly abstract code away, then all is well.